.ci/.e2e-tests.yaml

```diff
@@ -62,3 +62,9 @@ SUITES:
         tags: "integrations && redisenterprise"
       - name: "vSphere"
         tags: "integrations && vsphere"
+  - suite: "kubernetes-autodiscover"
+    platforms:
+      - "ubuntu-18.04"
+    scenarios:
+      - name: "kubernetes autodiscover with beats"
+        tags: "kubernetes && autodiscover && beats"
```

Comment:
As a suggestion trying to speed up the build, you can run the 3 feature files in parallel in three different workers. Simply explicitly tag each feature file with its own annotation (i.e. `@filebeat`, `@heartbeat`, `@metricbeat`), and create a new `- suite` section in this YAML file.

Reply:
Done.